packages/apps/src/initSettings.ts

```diff
@@ -70,10 +75,22 @@ function getApiUrl (): string {
       : 'ws://127.0.0.1:9944'; // nothing found, go local
 }
 
+function migrateStoringAccountStorage () {
+  const localStorageStoragingVersion = store.get('localStorageVersion') as number || 0;
```

Comment:
Yes, good point, done

Reply:
> Theoretically local storage could return something different than number

@youPickItUp I had the same remark, but since: 1. the `store` library parses the data from local storage and 2. the assertions are done everywhere else as well, I reckoned this is fine enough for this repo and adding validation looks like a more systemic thing to do here.

But generally, absolutely - `unknown` is 100% more type safe, so let's go for it if that's the will :)